spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

```diff
@@ -336,14 +339,20 @@ function startFlushUpgradeHandshake(
     currentChannel.state = desiredChannelState
     currentChannel.flushState = FLUSHING
 
-    // if there are no in-flight packets on our end, we can automatically go to FLUSHCOMPLETE
-    if pendingInflightPackets(portIdentifier, channelIdentifier) == nil {
-        currentChannel.flushState = FLUSHCOMPLETE
-    }
+    upgradeTimeout = getUpgradeTimeout(currentChannel.portIdentifier, currentChannel.channelIdentifier)
```

Comment:
Maybe I am missing something, but I cannot find where the upgrade timeout for the chain executing this function is stored...